server/store/token.go

```diff
@@ -16,6 +16,7 @@ type SQLTokenStore struct {
 type TokenStore interface {
 	Save(token *model.Token) error
 	Get(token string) (*model.Token, error)
+	GetLastToken() (*model.Token, error)
```

Comment:
Let's get rid of this. It's not clear why would we need last token and in the future tests when we create different tokens this will not work anyway.